Description: Use debian tools
 Tweak Makefile to use tools packaged by Debian
 instead of bundled with the source package
Author: dod
Applied-Upstream: NA
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 SHELL := /bin/bash
 
-CLOSURE_COMPILER=java -jar tools/closure-compiler/compiler.jar \
+CLOSURE_COMPILER=closure-compiler \
 	      --warning_level VERBOSE \
 	      --language_in ECMASCRIPT5 \
 	      --compilation_level ADVANCED_OPTIMIZATIONS \
@@ -10,8 +10,7 @@
 # to UTF-8 sequences instead of being emitted as ASCII.
 # This makes the resulting JavaScript less portable.
 
-YUI_COMPRESSOR=java -jar tools/yui-compressor/yuicompressor-2.4.4.jar \
-	      --charset UTF-8
+YUI_COMPRESSOR=yui-compressor --charset UTF-8
 
 TAR_ROOT=distrib/google-code-prettify
 
@@ -44,16 +43,12 @@
 	      | grep -v total; \
 	done
 	@$(CLOSURE_COMPILER) --js src/prettify.js \
-	    --externs tools/closure-compiler/console-externs.js \
-	    --externs tools/closure-compiler/amd-externs.js \
 	    --define IN_GLOBAL_SCOPE=true \
 	    --output_wrapper='!function(){%output%}()' \
 	    > $(TAR_ROOT)/prettify.js
 	@wc -c src/prettify.js $(TAR_ROOT)/prettify.js \
 	    | grep -v total
 	@$(CLOSURE_COMPILER) --js src/run_prettify.js \
-	    --externs tools/closure-compiler/console-externs.js \
-	    --externs tools/closure-compiler/amd-externs.js \
 	    --define IN_GLOBAL_SCOPE=false \
 	    --output_wrapper='!function(){%output%}()' \
 	    > $(TAR_ROOT)/run_prettify.js
